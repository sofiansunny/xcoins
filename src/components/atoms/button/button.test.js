import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import ButtonView from '.';
describe('SomeComponent component', () => {
  it('Shallow rendering', () => {
    const props = {
      onPress: () => {},
      title: '',
      isDisabled: false,
    };
    const wrapper = shallow(<ButtonView {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
