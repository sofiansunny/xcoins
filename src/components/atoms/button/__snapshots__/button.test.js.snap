// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent component Shallow rendering 1`] = `
<TouchableOpacity
  activeOpacity={0.7}
  onPress={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "#d63963",
        "borderRadius": 5,
        "elevation": 8,
        "paddingHorizontal": 12,
        "paddingVertical": 10,
      },
      Object {},
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "alignSelf": "center",
          "color": "#ffffff",
          "fontSize": 18,
          "textTransform": "uppercase",
        },
        Object {},
      ]
    }
  />
</TouchableOpacity>
`;
